src/legacy/core_plugins/tests_bundle/public/index.scss

```diff
@@ -0,0 +1,9 @@
+@import 'src/legacy/ui/public/styles/styling_constants';
+
+// vis_type_vislib UI styles are imported here for running karma Browser tests
+// should be somehow included through the "vis_type_vislib" plugin initialization
+@import '../../../../plugins/vis_type_vislib/public/index';
+
+// Visualization styles are imported here for running karma Browser tests
+// should be somehow included through the "visualizations" plugin initialization
+@import '../../../../plugins/visualizations/public/index';
```

Comment:
This is fine, though you probably don't have to repeat this same comment. Just add one at the very top of the file that explains the necessity of the file itself.

Reply:
@cchaos you are right, done.